### ISSUE TITLE
Question about voxels in training data

### ISSUE THREAD

**Issue by dmitrykudinov** [facebookresearch/meshrcnn#45]
Hi,

first, thank you for sharing your code!

I have a question about the need for voxelized meshes in the training data. If I understand this correctly, the voxels in training data are used to minimize the  voxel prediction loss. And the shapenet training actually relies on the precalculated model.binvox files. 

But the Pix3d training set does not have the model.binvox files... What happens then in case of pix3d - does the voxel head not get trained? or voxelization of the training Pix3D meshes happens on the fly?

**Comment by gkioxari** [comment]
Pix3D also provides voxels but in a .mat format and not in a .binvox format. You can see this in the data loaders for pix3D

https://github.com/facebookresearch/meshrcnn/blob/ab0762f87bfd4cbdf4560982c5c33832a445380b/meshrcnn/data/datasets/pix3d.py#L128

**Comment by dmitrykudinov** [comment]
Got it, thank you!
